PWGJE/EMCALJetTasks/AliFJWrapper.h

```diff
@@ -143,7 +143,9 @@ class AliFJWrapper
   void SetEventSub(Bool_t b) {fEventSub = b;}
   void SetMaxDelR(Double_t r)  {fMaxDelR = r;}
   void SetAlpha(Double_t a)  {fAlpha = a;}
-
+  void SetRecursiveDepth(Int_t n) {fRecursiveDepth = n;}
+  void SetZCut(Double_t zCut) {fZcut = zCut;}
+  void SetBeta(Double_t beta) {fBeta = beta;}
```

Comment:
I dug a little but can't find the answer yet, so in case you found out - we've e.g. varied the zcut in other measurements, so how did we use this in the past?

Reply:
I have no clue, I also try to find it. I guess everyone did it with using manual code. Also, the AliEmcalJetUtilitySoftDrop class is, in my opinion, not really useful for the SoftDrop analyses I found (at least it is not useful for my analysis, since it does not update the jet content - a second MR will follow after this one adding this functionality)

Reply:
Indeed, I implement it by hand in my analysis. Thanks for the info - it sounds like for your analysis your interested in grooming vs the usual tagging mode. Okay, let's see what comes of the other review. Perhaps the existing use of this class is actually pretty limited